src/main/java/com/easypost/model/Error.java

```diff
@@ -1,45 +1,46 @@
 package com.easypost.model;
 
+import java.util.List;
+
 public final class Error {
-    private String field;
     private String message;
-    private String suggestion;
     private String code;
+    private List<Error> errors;
 
     /**
-     * Get the error code.
+     * Get the errors from an Error object.
      *
-     * @return code
+     * @return errors
      */
-    public String getCode() {
-        return code;
+    public List<Error> getError() {
```

Comment:
Issue: This must be plural since it's a list of multiple errors

```suggestion
    public List<Error> getErrors() {
```